src/mlpack/tests/cv_test.cpp

```diff
@@ -164,6 +165,33 @@ BOOST_AUTO_TEST_CASE(MSETest)
   BOOST_REQUIRE_CLOSE(MSE::Evaluate(lr, data, responses), expectedMSE, 1e-5);
 }
 
+/**
+ * Test the R squared metric (R2Score).
+ */
+BOOST_AUTO_TEST_CASE(R2ScoreTest)
+{
+  // Making two points that define the linear function f(x) = x - 1
+  arma::mat trainingData("0 1");
+  arma::rowvec trainingResponses("-1 0");
+
+  LinearRegression lr(trainingData, trainingResponses);
+
+  // Making five responses that are the output of regression function f(x)
+  // with some responses having a slight deviation of 0.005
+  // Mean Responses = (1 + 2 + 3 + 6 + 8)/5 = 4
+  arma::mat data("2 3 4 7 9");
+  arma::rowvec responses("1 2.005 3 6.005 8.005");
+  
+  double ss_reg = (0 * 0 + 0.005 * 0.005 + 0 * 0
+  			 + 0.005 * 0.005 + 0.005 *0.005);
+  double ss_tot = ((1 - 4) * (1- 4) + (2.005 - 4) *
+  		(2.005 - 4) + (3 - 4) * (3 - 4) + (6.005 - 4) *
+  		(6.005 - 4) + (8.005 - 4) * (8.005 -4));
```

Comment:
We generally use either two tabs or align with the above statement. Could you change the indentation here to match that. Thanks.